sdk/go/common/apitype/service.go

```diff
@@ -31,8 +31,9 @@ const (
 	// via the PatchUpdateCheckpointDeltaRequest API to save on network bytes.
 	DeltaCheckpointUploadsV2 APICapability = "delta-checkpoint-uploads-v2"
 
-	// Indicates that the service backend supports stack bulk encryption.
-	BulkEncrypt APICapability = "bulk-encrypt"
+	// Indicates that the service backend supports batch encryption.
+	// These names don't match due to a historical rename for user consistency.
+	BatchEncrypt APICapability = "bulk-encrypt"
```

Comment:
Can the capability be renamed? We're not actively using this yet?

Reply:
Hmm, I guess so. It's already live in production, but I guess nothing should be depending on it's current value yet.

Reply:
Ugh, but actually, we need to merge here first before we can update the service ... will just change in this PR then update along with the types after.